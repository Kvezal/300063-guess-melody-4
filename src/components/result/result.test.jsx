import React from "react";
import render from "react-test-renderer";
import Enzyme, {shallow} from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import Result from "./result";

Enzyme.configure({
  adapter: new Adapter(),
});

describe(`ResultComponent`, () => {
  test(`should render component`, () => {
    const tree = render
      .create(<Result>Test</Result>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test(`should have result section`, () => {
    const resultComponent = shallow(<Result>Test</Result>);
    const result = resultComponent.find(`section.result`);
    expect(result).toHaveLength(1);
  });
});

