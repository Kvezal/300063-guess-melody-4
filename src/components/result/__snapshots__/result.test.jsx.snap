// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultComponent should render component 1`] = `
<section
  className="result"
>
  <div
    className="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  Test
</section>
`;
